### PR TITLE
[xcode11] [EventKit] bindings for all platforms

### PR DESCRIPTION
**Pull request by joconte**
Finally a lot of noise but not that much to change as I understood.

### CODE REVIEW

File: src/EventKit/EKEnums.cs [hunk 1]
```diff
@@ -75,6 +75,16 @@ public enum EKErrorCode : long {
...
+		[iOS (13,0), Mac (10,15)]
+		InvalidInviteReplyCalendar,
+		[iOS (13,0), Mac (10,15)]
+		NotificationsCollectionFlagNotSet,
+		[iOS (13,0), Mac (10,15)]
+		SourceMismatch,
+		[iOS (13,0), Mac (10,15)]
+		NotificationCollectionMismatch,
+		[iOS (13,0), Mac (10,15)]
+		NotificationSavedWithoutCollection,
```
Comment (joconte) [inline src/EventKit/EKEnums.cs]: I don't know which version of ```Watch```I should put.

**Comment by spouliot** [comment]
build

File: src/EventKit/EKEnums.cs (hunk 1 above)
Comment (spouliot) [reply src/EventKit/EKEnums.cs]: since this is added in Xcode 11 then it's the watchOS SDK version that was provided, i.e. 6.0

**Comment by rolfbjarne** [comment]
build

File: src/eventkit.cs [hunk 2]
```diff
@@ -508,16 +508,14 @@ interface EKEventStore {
...
-
-#if !MONOMAC
-		[NoWatch]
+		
+		[NoWatch, Mac (10,14)]
 		[Export ("saveEvent:span:error:")]
 		bool SaveEvent (EKEvent theEvent, EKSpan span, out NSError error);
 
-		[NoWatch]
+		[NoWatch, Mac (10,14)]
```
Comment (spouliot) [inline src/eventkit.cs]: Tests, run on 10.14, failed - that shows that this did not exists back then
Please update to `[Mac (10,15)]`

File: src/eventkit.cs [hunk 3]
```diff
@@ -508,16 +508,14 @@ interface EKEventStore {
 
 		[Export ("defaultCalendarForNewEvents"), NullAllowed]
 		EKCalendar DefaultCalendarForNewEvents { get;  }
-
-#if !MONOMAC
-		[NoWatch]
+		
+		[NoWatch, Mac (10,14)]
```
Comment (spouliot) [inline src/eventkit.cs]: Tests, run on 10.14, failed - that shows that this did not exists back then
Please update to `[Mac (10,15)]`

File: src/eventkit.cs (hunk 2 above)
Comment (joconte) [reply src/eventkit.cs]: Come on Apple.. ahah thanks

File: src/EventKit/EKEnums.cs [hunk 4]
```diff
@@ -75,6 +75,16 @@ public enum EKErrorCode : long {
 		ProcedureAlarmsNotMutable,
 		EventStoreNotAuthorized,
 		OSNotSupported,
+		[iOS (13,0), Mac (10,15), Watch (6,0)]
```
Comment (dalexsoto) [inline src/EventKit/EKEnums.cs]: `EKErrorCode` is an error enum so it is unlikely that you will have to use its values as input so you can remove all the availability information (`[iOS (13,0), Mac (10,15), Watch (6,0)]`) from all the new enum members.

**Comment by dalexsoto** [comment]
Build

**Comment by spouliot** [comment]
Thank again for the contribution!

**Pull request merged**